WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController.swift

```diff
@@ -1110,8 +1112,26 @@ extension NotificationsViewController: WPTableViewHandlerDelegate {
             selectFirstNotificationIfAppropriate()
         }
     }
-}
 
+    private static func accessibilityHint(for note: Notification) -> String? {
```

Comment:
Out of curiosity, why declare this as static?

Reply:
Thanks for asking! 

I wanted to explicitly declare that this is a pure utility function and shouldn't rely on anything in `NotificationsViewController`. If it was an instance method, somebody (like me, especially) might be tempted to use one of the `NotificationsViewController`'s properties when they modify this in the future. It should also help in debugging because we'll know that this doesn't use any state aside from `Notification`. 

Do you have some suggestions on how to make this better though? How do you usually implement things like this? I thought about adding a function at the bottom of the file but I wasn't sure if that's an accepted way.

Reply:
@shiki got it! Except for the protocol we use to make a type `Accessible` there's no specific rule for it. I think it depends mostly by the type and its design/context. So feel free to improve the `Accessible` protocol if you want or to find a better way to manage the accessibility in WP 😊